sky/serve/serve_utils.py

```diff
@@ -915,6 +941,15 @@ def terminate_services(cls, service_names: Optional[List[str]],
         ]
         return cls._build(code)
 
+    @classmethod
+    def terminate_replica(cls, service_name: str, replica_id: int,
+                          purge: bool) -> str:
+        code = [
+            f'msg = serve_utils.terminate_replica({service_name!r}, '
+            f'{replica_id}, {purge})', 'print(msg, end="", flush=True)'
+        ]
```

Comment:
Can we add some version checking here and dont execute if we find the version of controller is outdated? We should also return error message here.